tutorials/tutorial11_traffic_lights.ipynb

```diff
@@ -310,14 +284,68 @@
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Actuated Baseline Traffic Lights"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We have developed an actuated traffic light \"baseline\" that can be used for any experiments on a grid. This baseline uses actuated traffic lights (section 4), and has been fine-tuned on many iterations of experiments with varying parameters. The actual parameters are located in the `TrafficLightParams` class under the getter function `actuated_default()`. For reference, these values are:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tl_type = \"actuated\"\n",
+    "program_id = 1\n",
+    "max_gap = 3.0\n",
+    "detector_gap = 0.8\n",
+    "show_detectors = True\n",
+    "phases = [{\"duration\": \"31\", \"minDur\": \"8\", \"maxDur\": \"45\", \"state\": \"GrGr\"},\n",
+    "        {\"duration\": \"6\", \"minDur\": \"3\", \"maxDur\": \"6\", \"state\": \"yryr\"},\n",
+    "        {\"duration\": \"31\", \"minDur\": \"8\", \"maxDur\": \"45\", \"state\": \"rGrG\"},\n",
+    "        {\"duration\": \"6\", \"minDur\": \"3\", \"maxDur\": \"6\", \"state\": \"ryry\"}]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To see the actuated baseline traffic lights in action, simply initialize the TrafficLightParams class with the `baseline` argument set to `True`, and pass it into the `additional_net_params`. Nothing else needs to be done; no traffic lights need to be added. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tl_logic = TrafficLightParams(baseline=True)\n",
+    "additional_net_params = {\"grid_array\": grid_array, \n",
+    "                         \"speed_limit\": 35,\n",
+    "                         \"horizontal_lanes\": 1, \n",
+    "                         \"vertical_lanes\": 1,\n",
+    "                         \"traffic_lights\": True, \n",
+    "                         \"tl_logic\": tl_logic}"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## 6. Controlling Your Traffic Lights via RL\n",
-    "This is where we switch from the `grid.py` experiment script to `green_wave.py`. \n",
     "\n",
-    "To control traffic lights via RL, no `tl_logic` element is necessary. This is because rllab is controlling all the parameters you were able to customize in the prior sections. Your `additional_net_params` should look something like this: "
+    "This is where we switch from the non-RL experiment script to the RL experiment. \n",
+    "\n",
+    "To control traffic lights via RL, no `tl_logic` element is necessary. This is because rllib is controlling all the parameters you were able to customize in the prior sections. The `additional_net_params` should look something like this: "
```

Comment:
Hmm, don't think we should hardcode in RLlib. I think it's better to say that the RL agent is controlling the parameters.